### DISCUSSION TITLE
Internet explorer issue on unmounting component

### DISCUSSION THREAD

**Discussion by 1IvI1** [ValentinH/react-easy-crop#215] [Q&A] [answered]
After I unmount component this error is displayed on IE11
![image](https://user-images.githubusercontent.com/53253811/110774629-ecb44400-8255-11eb-8055-ef3cacc88c82.png)

**Comment by ValentinH** [comment, marked as answer]
This is weird because we are using this component in production and we support IE11 without any issues being reported to our Error monitoring tools.

Are you sure this is coming from this component?

**Reply by 1IvI1** [reply]
> This is weird because we are using this component in production and we support IE11 without any issues being reported to our Error monitoring tools.
> 
> Are you sure this is coming from this component?

Found that problem was in componentWillUnmount, where this.styleRef.remove() is being called.
![image](https://user-images.githubusercontent.com/53253811/110779052-0310ce80-825b-11eb-8df9-99c6bb543602.png)

**Fix:**
1) Added "disableAutomaticStylesInjection" prop on Cropper
![image](https://user-images.githubusercontent.com/53253811/110779402-669afc00-825b-11eb-90bf-85cc4a967d5b.png)
2) Imported styles for Cropper in my component

**Reply by ValentinH** [reply]
Ah yes, good point. We are not using this one that's why. I guess it's due to remove 'ot being available in IE11. I'll check for a workaround.

**Reply by ValentinH** [reply]
This is fixed in v3.3.2
